org.eclipse.wildwebdeveloper/pom.xml

```diff
@@ -204,6 +204,8 @@
 								<arg>--only=prod</arg>
 								<arg>--no-bin-links</arg>
 								<arg>--ignore-scripts</arg>
+								<arg>--install-strategy=hoisted</arg>
+								<arg>--force</arg>
```

Comment:
Make sure dependencies are always fresh, without this tests fail on github but pass jenkins

Reply:
Force also will ignore platform requirements, so will be possible to add fsevents for mac users ;)